code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -254,7 +254,7 @@ GLOBAL_LIST_INIT(titaniumglass_recipes, list(
 //////////////////////////////
 /obj/item/stack/sheet/titaniumglass
 	name = "titanium glass"
-	desc = "A glass sheet made out of a titanium-silicate mixture."
+	desc = "A glass sheet made out of a titanium-silica mixture."
```

Comment:
This isn't really part of the fix, `silicate` is a legit naming convention https://www.britannica.com/science/silicate-mineral

Reply:
Silica is a specific material (Silicon dioxide, SiO2), silicate a is a broad category of substances made up of silicate anions (generally with a skeleton of SiO4 units), most of which are useless for glassmaking.

I mostly made this change because currently there is a mix of "silica" and "silicate" in the descriptions, but the fact of the matter is it's also more correct.

Reply:
So would "A glass sheet made out of titanium-silicate" be more accurate?

Reply:
This does flow slightly better.

Reply:
It flows nicer and as it turns out ti-si *is* optically clear, so yeah, I'll tweak that.